src/gemdat/volume.py

```diff
@@ -335,15 +348,10 @@ def to_structure(
 
         return structure
 
-    def to_probability(self, ) -> Volume:
-        """Normalize the volume to use it as a probability."""
-        self.data = self.data / self.data.sum()
-        return self
-
     def get_free_energy(
```

Comment:
Would It make sense for this method to **not** return a new `Volume`, but instead just adding the 'free_energy' key to self.data?
This would require changing a bit the .ipynb , but I don't think that the user needs to access Volume.data['free_energy']  while the `free_energy_graph` is available